pkg/nfd-master/nfd-master.go

```diff
@@ -249,6 +239,17 @@ func (m *nfdMaster) Run() error {
 			if err := m.configure(m.configFilePath, m.args.Options); err != nil {
 				return err
 			}
+
+			// restart NFD API controller
+			if m.nfdController != nil {
+				m.nfdController.stop()
```

Comment:
Maybe add a log line that we're stopping the controller(?) As we're also logging it being started